ci/pytest_skips.ini

```diff
@@ -29,3 +29,66 @@ addopts =
           --deselect jax/tests/qdwh_test.py::QdwhTest::testQdwhWithUpperTriangularInputAllOnes
           --deselect jax/tests/qdwh_test.py::QdwhTest::testQdwhWithDynamicShape
           --deselect jax/tests/qdwh_test.py::testQdwhWithRandomMatrix
+#failed in rocm-v0.7.1
+          --deselect jax/tests/sparsify_test.py::SparsifyTest::testSparseMul
+          --deselect jax/tests/sparsify_test.py::SparsifyTest::testUnaryOperationsNonUniqueIndices
+          --deselect jax/tests/sparsify_test.py::SparsifyTracerTest::testSparseDiv
```

Comment:
Also is pytest able to find these tests properly to skip them? Typically if you are not giving the full test name (such as jax/tests/pallas/ops_test.py::OpsTest::test_approx_tanh0, pytest does not find the module name properly). If you want to completely skip all the variations of jax/tests/pallas/ops_test.py::OpsTest::test_approx_tanh tests you will need to use the -k argument when running pytest. I think for skip.ini, this behavior may be same or different. 

So as a summary, I want to learn if these were verified as skipped in the tests.